benchmarks/dynamo/common.py

```diff
@@ -3170,7 +3170,24 @@ def process_entry(rank, runner, original_dir, args):
         )(runner, args, original_dir)
 
 
+def run_startup_script():
+    """
+    Follow https://docs.python.org/3/tutorial/appendix.html so people can run
+    a startup script using PYTHONSTARTUP environment variable.
+
+    Handy to use for overriding config, logging level etc.
+    """
+    import os
+
+    filename = os.environ.get("PYTHONSTARTUP")
+    if filename and os.path.isfile(filename):
+        with open(filename) as fobj:
+            startup_file = fobj.read()
+        exec(startup_file)
+
+
```

Comment:
This should be a different PR.  I'm a bit skeptical of this idea... why can't you just set env variables in bash scripts?

Reply:
yea, in most cases we can use a shell script wrapper. The slight advantage of the python startup script is

1. we can override some configs that does not have corresponding env variables yet. (although we can add environment variables for them as an alternative)
2. I feel python script more flexible and easier to manage.

But those are not very big advantage TBH, so I'm fine to drop it